packages/paste-core/components/form/src/FormTextArea.tsx

```diff
@@ -66,6 +67,7 @@ const FormTextArea = React.forwardRef<HTMLTextAreaElement, FormTextAreaProps>(
           aria-invalid={hasError}
           aria-readonly={readOnly}
           {...props}
+          async
```

Comment:
I understand the `async` concept, but am curious what adding `async` here does for the styled component?

Reply:
I'm guessing it has something to do with adding the arrow function on line 28.

Reply:
No, it's actually a little more criptic than that. It's part of the API of the `TextareaAutosize` component https://github.com/buildo/react-autosize-textarea#props.

But because I'm using a styled version of it, you can't immediately tell it's actually part of that components API.

Reply:
Ok. I see that now. Thanks for the clarification. I pulled down locally and everything looks like its working. I like how it still adheres to our max-height.